src/chi-vue/src/constants/types.ts

```diff
@@ -186,6 +186,25 @@ export type PaginationSizes = typeof PAGINATION_SIZES[number];
 //#endregion
 
 //#region Save View
+export interface SaveViewConfig {
+  active: boolean;
+  results: string;
+  default?: boolean;
+  id?: string;
+  mode?: SaveViewModes;
+  title?: string;
+  saveButtonDisabled?: boolean;
+  readonly?: boolean;
+  label?: {
+    delete?: string;
+    input?: string;
```

Comment:
```suggestion
    label?: string;
```

Also will move it under input properties